src/viewer.ts

```diff
@@ -0,0 +1,27 @@
+import { underline, yellow } from 'colorette'
+import { eventHandler, sendRedirect } from 'h3'
+import { addDevServerHandler, isNuxt2, isNuxt3, useLogger, useNuxt } from '@nuxt/kit'
+import { withTrailingSlash, withoutTrailingSlash, joinURL } from 'ufo'
+import type { TWConfig } from './types'
+
+export default async function (twConfig: Partial<TWConfig>, nuxt = useNuxt()) {
+  // const { withTrailingSlash, withoutTrailingSlash, joinURL } = await import('ufo')
```

Comment:
My understanding from this is that the dynamic import is for treeshaking/bundle-size?

Although `ufo` is in the module dependencies as well as lots of Nuxt packages - I feel that it is negligible...

Reply:
Agree we can avoid having `await import('ufo')` since it is anyway on the server part